Hello.java

```diff
@@ -2,6 +2,6 @@
 {
    public static void main(String args[])
```

Comment:
Can you modify code to use args to pass "hello world" in as a parameter, then use parameter to print what you did?